optaplanner-core/src/test/resources/logback-test.xml

```diff
@@ -5,7 +5,7 @@
     <encoder>
       <!-- %L lowers performance -->
       <!--<pattern>%d{HH:mm:ss.SSS} [%t] %-5p %L%n  %m%n</pattern>-->
-      <pattern>%d{HH:mm:ss.SSS} [%t] %-5p %m%n</pattern>
+      <pattern>%d{HH:mm:ss.SSS} [%-4.4t] %-5p %m%n</pattern>
```

Comment:
Good idea. I 'd like to keep this consistent between all logback-test.xml's and even the examples logback.xml and 4 chars was a bit too abbreviated.
After some experimentation it turns out that 12 chars was long enough to be clear and still reasonable in length, so I 've pushed that to all logback xml files.